manim/utils/file_ops.py

```diff
@@ -77,3 +78,22 @@ def open_file(file_path, in_browser=False):
             raise OSError("Unable to identify your operating system...")
         commands.append(file_path)
         sp.Popen(commands)
+
+
+def open_media_file(file_writer):
+    file_paths = []
+
+    if config["save_last_frame"]:
+        file_paths.append(file_writer.image_file_path)
+    if config["write_to_movie"] and not config["save_as_gif"]:
+        file_paths.append(file_writer.movie_file_path)
+    if config["save_as_gif"]:
+        file_paths.append(file_writer.gif_file_path)
+
+    for file_path in file_paths:
+        if config["show_in_file_browser"]:
+            open_file(file_path, True)
+        if config["preview"]:
+            open_file(file_path, False)
+
+        logger.info(f"Previewed File at: {file_path}")
```

Comment:
Issue : I don't think this line is correctly indented, since I guess this log is supposed to be displayed when a file is previewed.

Reply:
originally this function was only meant to preview, but I needed to add the other things because an old function got removed (internal).